scripts/au/Dockerfile

```diff
@@ -1,11 +1,11 @@
-FROM ubuntu:14.04
+FROM ubuntu:17.04
 
 RUN apt-get update -y && \
-    apt-get install -y git curl zip unzip parallel \
-                    postgresql-9.3 postgresql-9.3-postgis-2.1 \
+    apt-get install -y sudo git curl zip unzip parallel \
+                    postgresql-9.6 postgresql-9.6-postgis-2.3 postgis \
```

Comment:
* needed to install sudo for script to run
* upgrading postgis/postgis in line with gnaf-loader recommendation
* postgis is needed to provide shp2pgsql for gnaf-loader